launch/interact.go

```diff
@@ -66,13 +70,30 @@ func (d *Interact) Run(c *exec.Cmd, commands [][]string) error {
 	// Copy stdin to the pty and the pty to stdout.
 	go func() {
 		for _, v := range commands {
-			v = append(v, "\n")
-			_, _ = io.Copy(ptmx, strings.NewReader(strings.Join(v, " ")))
-		}
 
+			v := append(v, "\n")
+			command := strings.Join(v, " ")
+			if len(command) > maxByte {
```

Comment:
I think this outer if is unnecessary.

Reply:
Thank you for pointing that out.
You're certainly right:+1: